services/static-webserver/client/source/class/osparc/ui/form/EditLabel.js

```diff
@@ -96,102 +104,112 @@ qx.Class.define("osparc.ui.form.EditLabel", {
       apply: "_applyEditable"
     }
   },
+
   members: {
-    __label: null,
-    __input: null,
     __labelWidth: null,
     __loadingIcon: null,
+
+    _createChildControlImpl: function(id) {
+      let control;
+      switch (id) {
+        case "text":
+          control = new qx.ui.basic.Label().set({
+            appearance: "editlabel-label"
+          });
+          // make it clear that the field is editable
+          control.getContentElement().setStyles({
+            "border-radius": "4px"
```

Comment:
We could use the rounded decoration rather than setStyles. Probably slightly better for qooxdoo?

Reply:
Thanks